drivers/hopwa_caper/app/models/hopwa_caper/generators/fy2024/enrollment_filters/income_benefit_level_filter.rb

```diff
@@ -0,0 +1,40 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HopwaCaper::Generators::Fy2024::EnrollmentFilters
+  IncomeBenefitLevelFilter = Struct.new(:label, :type, keyword_init: true) do
+    def apply(scope)
+      cond = HopwaCaper::Enrollment.group(:report_household_id).having('MAX(percent_ami) = ?', code)
```

Comment:
Calling MAX here will return `99` if Data Not Collected for some clients, does it matter that there might be a 4, or 3, etc. that are then being ignored?  Maybe this should preference collected values somehow?

Reply:
oh good catch! I'll just filter to pct_api <= 4